data/regen.sh

```diff
@@ -14,3 +14,7 @@ rm -fv data/system/maps/huge_seattle.bin data/input/raw_maps/huge_seattle.bin da
 cargo run --release --bin game -- --prebake
 cargo run --release --bin game -- --smoketest
 cargo run --release --bin game -- --check_proposals
+
+# This is cheap enough to do before every single commit, but since we don't
+# have presubmit tests, it's fine to just run it here.
```

Comment:
In some projects I check in some suggested git hooks. e.g.:

```
scripts/git_hooks/precommit
scripts/git_hooks/postcommit
```

Then locally developers can do something like
```
for file in ls scripts/git_hooks/*; do
    ln -s $file .git/hooks
done
```

Honestly, it's kind of janky, but it's the best I can suggest. 😅

Reply:
That's a nifty trick. :P  I'll try this out and see how it works. I use `git commit --amend` constantly in my local workflow and don't want to trigger it then. There's also a very low risk of accidentally breaking turn generation without realizing it; it's pretty obvious when a PR is touching relevant code and requires map data regeneration.

Reply:
I guess something like this should usually work:

https://stackoverflow.com/a/56955369/353178

```
IS_AMEND=$(ps -ocommand= -p $PPID | grep -e '--amend');

if [ -n "$IS_AMEND" ]; then
  return;
fi

do_the_thing
```

Reply:
I'm also a heavy ammender: `alias gca='git commit --amend'` =)